ivy/container/elementwise.py

```diff
@@ -1969,9 +1991,31 @@ def cosh(
         Returns
         -------
         ret
-            an container containing the hyperbolic cosine of each element in ``self``.
+            an container containing the hyperbolic cosine of each element in ``self``. 
             The returned container must have a floating-point data type determined by
             :ref:`type-promotion`.
+
+        Examples
+        --------
+        With :code:`ivy.Container` input:
+
+        >>> x = ivy.Container(a=ivy.array([-1, 0.23, 1.12]), b=ivy.array([1, -2, 0.76]))
+        >>> y = x.sinh()
```

Comment:
you must have meant x.cosh(), please fix this so that docstring tests are passed